src/test/java/create_task_use_case/CreateTaskInteractorTest.java

```diff
@@ -0,0 +1,19 @@
+package create_task_use_case;
+
+import entities.TaskFactory;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreateTaskInteractorTest {
+    private final static CreateTaskOutputBoundary createTaskPresenter = new CreateTaskPresenter();
+    private final static CreateTaskDataAccess createTaskDataAccess = new CreateTaskDataAccess();
+    private final static TaskFactory taskFactory = new TaskFactory();
+    @Test
+    public void createTest(){
```

Comment:
The above comment from Raghav needs to be addressed (possibly through existsByTitle as I mentioned in response to an earlier comment). 
This test case should be renamed testCreate to align with our established test case naming conventions.

Reply:
Done